docs/advanced/test-command.md

```diff
@@ -64,6 +68,81 @@ charon alpha test beacon \
 ### Run with Docker
 
 ```shell
-docker run obolnetwork/charon:v1.0.0 alpha test beacon \
+docker run obolnetwork/charon:v1.1.0 alpha test beacon \
   --endpoints="https://ethereum-holesky-beacon-api.publicnode.com,https://ethereum-sepolia-beacon-api.publicnode.com"
 ```
+
+## Test your validator client
+
+Run tests towards your validator client, to evaluate its effectiveness for a Distributed Validator cluster.
+
+Default endpoint for validator and port is used at `127.0.0.1:3600`. This can be changed by supplying different endpoint to the `--validator-api-address` flag.
+
+### Pre-requisites
+
+- Running validator client towards which tests will be executed.
+
+### Run
+
+```shell
+charon alpha test validator
+```
+
+### Run with Docker
+
+```shell
+docker run obolnetwork/charon:v1.1.0 alpha test validator
+```
+
+## Test MEV relay
+
+Run tests towards MEV relay(s), to evaluate its effectiveness for a Distributed Validator cluster. If a MEV relay is configured for Charon, it is of utmost importance that the connection to it is fast. If not, the chance of missing a block proposal raises significantly, because of a slow building of the block from the MEV.
```

Comment:
> If a MEV relay is configured for Charon

Technically Charon doesn't ever interact with MEV relay, rather mev-boost does, and in turn the VC speaks with mev-boost.